packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -395,6 +395,7 @@ export default mixins(
 						this.$showError(error, 'Problem deleting the workflow', 'There was a problem deleting the workflow:');
 						return;
 					}
```

Comment:
this will set state dirty to false even if the workflow is not deleted. can we handle the scenario when delete is cancelled

Reply:
Hey @mutdmour , for both cases when canceling the delete or when an error trigger in the delete request, the state dirty stays doesn't change as the `return` statements prevent the flow from reaching the `store.commit` statement.

Reply:
I see thank you. makes sense